configen/OptionsParser.swift

```diff
@@ -54,14 +53,13 @@ final class OptionsParser {
 
     return plistDictionary
     }()
-
-  lazy var hintsDictionary: [String: String] = { [unowned self] in
+  
+  lazy var sortedHintsTuple: [(key: String, value: String)] = { [unowned self] in
```

Comment:
We aren't using the explicit labels anywhere so perhaps we can drop this (or change existing code to use it)

Reply:
Changed to use a typealias. Let me know what you think 👍

Reply:
Nice 👌